test/integration/apiserver/admissionwebhook/timeout_test.go

```diff
@@ -427,7 +427,7 @@ func newTimeoutWebhookHandler(recorder *timeoutRecorder) http.Handler {
 			return
 		}
 
-		timeout, err := time.ParseDuration(r.URL.Query().Get("timeout"))
+		timeout, _ := time.ParseDuration(r.URL.Query().Get("timeout"))
```

Comment:
I think it'd be better to act on the error (e.g. as above `http.Error(w, err.Error(), 400)`) to help people find test problems faster.

Reply:
@lavalamp Sure . I will update it.Thanks